### DISCUSSION TITLE
YALMIPTEST giving an error

### DISCUSSION THREAD

**Discussion by akdas29** [yalmip/YALMIP#1301] [Problems using YALMIP] [answered]
Hi! I am trying to do a semidefinite programming(SDP) for that I have installed YALMIP-master from github. But when I run the yalmiptest command in the command window it is giving me the following errors. The code and the error messages are attached herewith. Please suggest me how to fix this issue. Thanks in advance.
 ```
for j=0 :1 :10
       q = sdpvar(1,1,'hermitian','real'); 
       q= j* 0.1;
       A1 = q*[1 0; 0 0] + ((1-q)/2)*eye(2); A2 = eye(2) - A1;
      B1 = q*(1/2)*[1 1; 1 1] + ((1-q)/2)*eye(2); B2 = eye(2) - B1;
       
d=2;

     m = sdpvar(1,1,'hermitian','real'); 
    G11 = sdpvar(d,d,'hermitian','complex'); 
    G12 = sdpvar(d,d,'hermitian','complex'); 
    G21 = sdpvar(d,d,'hermitian','complex'); 
    G22 = sdpvar(d,d,'hermitian','complex'); 
Cons = [G11+G12 == A1; G21+G22 == A2; G11+G21 == B1; G12+G22 == B2; G11+G12+G21+G22 == eye(2); ...
    G11 >= m*eye(2);G12 >= m*eye(2);G21 >= m*eye(2);G22 >= m*eye(2)];
    
    
ops = sdpsettings('solver','sdpt3','verbose',0) 

optimize(Cons,-real(m),ops) 
value(m)
value(q)

end
```
![Yalmip](https://github.com/yalmip/YALMIP/assets/136682744/c3a267ab-38c9-46f6-bcf9-4d83c1e8202c)

**Comment by johanlofberg** [comment]
The crash is telling you the file svec is broken somehow (that's a file in the sdpt3 distribution). Hence, you will have to check what's up with that file

```
K>> dbtype 1:12 svec

1     %******************************************************************
2     %% svec: compute the vector svec(M),
3     %%
4     %%   x = svec(blk,M,isspx);
5     %%*****************************************************************
6     %% SDPT3: version 4.0
7     %% Copyright (c) 1997 by
8     %% Kim-Chuan Toh, Michael J. Todd, Reha H. Tutuncu
9     %% Last Modified: 16 Sep 2004
10    %%*****************************************************************
11    
12      function x = svec(blk,M,isspx); 
```

**Reply by akdas29** [reply]
I have modified my svec.m file by yours. Still showing errors.
![Screenshot from 2023-06-20 12-39-41](https://github.com/yalmip/YALMIP/assets/136682744/359638af-8e70-40d3-b37d-ed315f8b62a3)

**Reply by johanlofberg** [reply]
You should not edit any file. You should confirm that it looks like it should (and that it is a file in the sdpt3 distribution and isn't some file lurking in your path in some other solver or something)

**Reply by akdas29** [reply]
That problem is solved but showing another error. It shows yalmiptime: NaN, solvertime: NaN

![Screenshot from 2023-06-22 21-39-58](https://github.com/yalmip/YALMIP/assets/136682744/2c64c312-8aa7-48c8-aa3b-876d5b380bb4)

**Comment by johanlofberg** [comment]
Did you click on the link?

**Comment by akdas29** [comment]
I have clicked the link and it does open a browser but I could not access any information from it as all of them are non-responsive.  The most annoying thing is every time the command window shows some file is corrupted and when I repair it and run the code again it shows that some another file is corrupted and it goes on. 

![Screenshot from 2023-06-23 11-34-49](https://github.com/yalmip/YALMIP/assets/136682744/b1b7dedc-228f-498a-a9b7-46446ee8987f)

**Reply by johanlofberg** [reply]
What os is that? It must be broken somehow. It is simply a link to https://yalmip.github.io/inside/debug/

However, the info line already tells you what the issue is, you have more broken files in sdpt

**Reply by akdas29** [reply]
I am using ubuntu 20.04. I guess there are multiple files crashed in the path 'sdpt3-master/Solver'. How can I download the whole folder 'Solver' from sdpt3 ?  I can download the files one by one but not all together.

I don't know why the same folder 'Solver' is working fine for my friend but when I taking it from her it is not working in my computer.

**Reply by johanlofberg** [reply]
Filer by file?! That sounds like whole working day of agony

Download and unpack the zip, or clone the repository
https://github.com/sqlp/sdpt3

**Reply by akdas29** [reply, marked as answer]
Thank you so much! The problem solved.  :)
